src/logic/equiv/transfer_instance.lean

```diff
@@ -383,11 +383,11 @@ end equiv
 
 namespace ring_equiv
 
-protected lemma local_ring {A B : Type*} [comm_ring A] [local_ring A] [comm_ring B] (e : A ≃+* B) :
-  local_ring B :=
+protected lemma local_ring {A B : Type*} [comm_semiring A] [local_ring A] [comm_semiring B]
```

Comment:
Why `protected`? Even if it is a special case of `of_surjective` one may want to use this.

Reply:
I think `protected` (remark: not `private`) here is intended to use the lemma with the fully specified name. Since it seems unlikely that we will refer to `ring_equiv.local_ring` as ` local_ring`, I think `protected` here is reasonable.

Reply:
You're totally right, I was thinking about `private`.